game/src/sandbox/speed.rs

```diff
@@ -595,9 +600,10 @@ impl State for TimeWarpScreen {
                     now.ampm_tostring(),
                     self.target.ampm_tostring()
                 )),
+                Line(format!("Elapsed simulation time: {}", elapsed_sim_time)),
```

Comment:
Maybe this was intentional? But I was confused by the phrasing. 

Is it useful to know how long since you've clicked the button?

Reply:
To clarify - this now shows how much time has been simulated, before it was showing how longer we'd been simulating.

Reply:
The wording was awful, it was intentionally meant to tell you how much real time had passed. But looking at this again, really the interesting thing is just the playback speed. How about we stop showing elapsed time so far entirely (simulation and real time) and just show speed? The elapsed sim time can be understood from the "now / target time" display